cea/analysis/senstivity_demand.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+"""
+===========================
+Sensitivity of demand_main.py
+
+This script uses the morris algorithm (morris 1991)(campologo 2011) and Sobol Algorithm Sltalli 20110
+to screen the most sensitive variables of a selection of parameters of the CEA.
+Th morris method serves to basic screening o finput variables and it is base on OAT
+The Sobol method serves for a complete sensitivity analysis of input variables. It is based on variance methods.
+
+===========================
+
+"""
+from __future__ import division
+
+from cea.demand import demand_main
+
+import multiprocessing as mp
+from SALib.analyze import sobol, morris
+from SALib.sample.saltelli import sample as sampler_sobol
+from SALib.sample.morris import sample as sampler_morris
+import pandas as pd
+import numpy as np
+import time
+
+__author__ = "Jimeno A. Fonseca"
+__copyright__ = "Copyright 2016, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Jimeno A. Fonseca"]
+__license__ = "MIT"
+__version__ = "0.1"
+__maintainer__ = "Daren Thomas"
+__email__ = "thomas@arch.ethz.ch"
+__status__ = "Production"
+
+# main
+def sensitivity_main(locator, weather_path, gv, output_parameters, groups_var, num_samples, method):
+    """
+    This function creates the sensitivity analysis problem for either sobol or morris methods.
+    It calculates the number of samples for the analysis and calls the CEA to run every sample.
+
+
+    :param locator: locator class
+    :param weather_path: path to weather file
+    :param gv: global variables class
+    :param output_parameters: list of output parameters to analyse
+    :param groups_var: list with group names of variables to analyse. it relates the database of
```

Comment:
I added a better explaination, but would leave the explanation of the database to the manual. is taht ok?